.storybook/preview.js

```diff
@@ -24,9 +24,9 @@ export const parameters = {
 };
 
 export const decorators = [
-  Story => (
+  story => (
     <VerticalCenter style={{alignItems: 'center', minHeight: '100vh', boxSizing: 'border-box', display: 'flex', justifyContent: 'center'}}>
-      <Story />
+      {story()}
```

Comment:
didn't we make this change originally because it was crashing storybook as you have it now?